next/components/social-share.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTranslation } from "next-i18next";
+import { useEffect, useState } from "react";
+
+import FacebookIcon from "../styles/icons/facebook.svg";
+import LinkedInIcon from "../styles/icons/linkedin.svg";
+import TwitterIcon from "../styles/icons/twitter.svg";
+
+export function SocialShare() {
+  const [pageUrl, setPageUrl] = useState<string>("");
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    const currentUrl = window.location.href;
+    setPageUrl(currentUrl);
+  }, [pageUrl]);
```

Comment:
`pageUrl` is added to the dependency array, but I don't think it's needed, right?